include/libpldm/base.h

```diff
@@ -131,8 +131,8 @@ struct pldm_msg_hdr {
 	uint8_t type : 6;	//!< PLDM type
 	uint8_t header_ver : 2; //!< Header version
 #elif defined(__BIG_ENDIAN_BITFIELD)
-	uint8_t header_ver : 2;	 //!< Header version
-	uint8_t type : 6;	 //!< PLDM type
+	uint8_t header_ver : 2; //!< Header version
+	uint8_t type : 6;	//!< PLDM type
 #endif
```

Comment:
can this change go as separate commit? any reason to include them in he present commit? 
or just a linter check change?

Reply:
Okay I am separating those two. First I will use this for linter and create a separate PR for User password support.